LORIS_indepth.html

```diff
@@ -670,8 +402,213 @@ <h2>LORIS - Github <font color="yellow"> - Open Source</font></h2>
                         <h2>LORIS - Github<font color="yellow"> Usage</font></h2>
```

Comment:
yellow -> orange if you can

Reply:
done - changed all yellow in the code to loris orange